lib/messagebird.js

```diff
@@ -17,13 +17,18 @@ var extend = Object.assign ? Object.assign : require('util')._extend;
  * @param {Integer} timeout
  * @return {Object}
  */
-module.exports = function (accessKey, timeout) {
+module.exports = function (accessKey, timeout, features) {
```

Comment:
There is also a type definition in this project where the possible values need to be added. You can find it in the `types` folder in the root of this project.

Reply:
I have added it to types and pushed the change.